Source/Core/FeatureDetection.js

```diff
@@ -199,6 +203,37 @@ define([
         return supportsImageRenderingPixelated() ? imageRenderingValueResult : undefined;
     }
 
+    var supportsWebpPromise;
+    function supportsWebp() {
+        // From https://developers.google.com/speed/webp/faq#how_can_i_detect_browser_support_for_webp
+        if (defined(supportsWebpPromise)) {
+            return supportsWebpPromise;
+        }
+
+        var webpDataUri = 'UklGRiIAAABXRUJQVlA4IBYAAAAwAQCdASoBAAEADsD+JaQAA3AAAAAA';
+        var image = new Image();
```

Comment:
Move this to below the edge check so you don't create it unless you are actually using it.

Reply:
I was going to suggest using `Resource.fetchImage` here since it would simplify the code, but it would most likely create a circular dependency since this is a pretty low level file.  So no action on you part, but just wanted to bring that idea up in case anyone was thinking the same thing.